app/services/library_search_service.rb

```diff
@@ -19,7 +19,7 @@ def to_a
   #
   # @return [Integer] the total number of results
   def total_count
-    result_media.total_count
+    algolia_result_media.length
```

Comment:
I think this will give the wrong number due to pagination; Algolia should probably give a _total results count_ somewhere in the response data